prime-dotnet-webapi/ViewModels/Health Authority Sites/_HealthAuthoritySiteListMappingProfile.cs

```diff
@@ -0,0 +1,15 @@
+using AutoMapper;
+
+using Prime.Models;
+
+namespace Prime.ViewModels.HealthAuthoritySites
+{
+    public class HealthAuthoritySiteListMappingProfile : Profile
+    {
+        public HealthAuthoritySiteListMappingProfile()
+        {
+            CreateMap<HealthAuthoritySite, HealthAuthoritySiteListViewModel>()
+                .ForMember(dest => dest.AdjudicatorIdir, opt => opt.MapFrom(src => src.Adjudicator.IDIR));
```

Comment:
This can be added to the existing `_HealthAuthoritySiteMappingProfile.cs`, you don't have to make a new profile.